libdnf/dnf-sack.cpp

```diff
@@ -376,6 +368,24 @@ dnf_sack_recompute_considered(DnfSack *sack)
         map_and(pool->considered, &pkg_includes_tmp);
         map_free(&pkg_includes_tmp);
     }
+}
+
+/**
+ * dnf_sack_recompute_considered:
+ * @sack: a #DnfSack instance.
+ *
+ * DOES SOMETHING.
```

Comment:
I know this was there before but... DO SOMETHING about it? :slightly_smiling_face:

Reply:
Same description is used 4x in the module.

Reply:
Resolved